foundation/plugins/org.jboss.tools.foundation.help.ui/.classpath

```diff
@@ -0,0 +1,8 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<classpath>
+	<classpathentry kind="con" path="org.eclipse.jdt.launching.JRE_CONTAINER/org.eclipse.jdt.internal.debug.ui.launcher.StandardVMType/JavaSE-1.7"/>
+	<classpathentry kind="con" path="org.eclipse.pde.core.requiredPlugins"/>
+	<classpathentry exported="true" kind="lib" path="lib/jboss-dmr-1.2.0.Final.jar"/>
```

Comment:
1.2.0 in as.management.wildfly
1.1.6 in as.management.as7
1,1,1 in ide.eclipse.as.dmr

These three are justified in that as.dmr is exposed to clients,  while the other two are internal details to the wf / as7 management implementations.  The different versions between as7 / wf management plugins is because those are what's required for the various remoting 

It's possible that 1.2.0 would work for all cases but an inspection of the API differences between them would be required.